DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi/Startup.cs

```diff
@@ -52,16 +52,15 @@ public void Configuration(IAppBuilder app)
 
             app.CreatePerOwinContext<ApplicationUserManager>(ApplicationUserManager.Create);
 
-            FluentValidationModelValidatorProvider
-                .Configure(config,
-                    provider => provider.ValidatorFactory = new FluentValidatorFactory(container));
+            //FluentValidationModelValidatorProvider

```

Comment:
Currently we have a bug with fluent validation, so Serhiy advised to disable Fluent Validation until it is fixed